build/build.rb

```diff
@@ -89,6 +89,10 @@ def verify_run(output)
     file_name = "#{name}-#{target}-#{build_label}-#{timestamp}-#{git_checkout.commit_sha}.#{FILE_TYPE[imgfac_target]}"
     destination = DESTINATION_DIRECTORY.join(file_name)
     $log.info `mv  #{source} #{destination}`
+    $log.info "Generating image checksums"
+    $log.info `cd #{DESTINATION_DIRECTORY}/; /usr/bin/sha256sum * >> SHA256SUM`
+    $log.info `cd #{DESTINATION_DIRECTORY}/; /usr/bin/gpg --batch --no-tty --passphrase-file /root/.gnupg/pass -b SHA256SUM`
+    $log.info `cd #{DESTINATION_DIRECTORY}/; cp /root/.gnupg/cfme_public.key . `
```

Comment:
@JPrause I suggest investigation using  FileUtils.cp in place of shelling out a cp command.